core/shared/src/engine.cpp

```diff
@@ -268,7 +285,7 @@ void Engine::Close()
 #ifdef _WIN32
 		processPath = "bin/updater.exe";
 #else
-		processPath = "lib/updater";
+		processPath = "bin/updater";
```

Comment:
@SlawekNowy Why change this to "bin/"? Linux binaries for pragma should usually be under "lib/"

Reply:
Save from the executables, which are usually on bin.
Updater is such an executable.